site/src/Controller/OrganizationController.php

```diff
@@ -215,6 +220,23 @@ public function joinAction($organizationName, OrganizationRepository $repository
         $member->setUser($user);
         $member->setPermission(OrganizationMember::PERMISSION_REQUEST);
         $em = $this->getDoctrine()->getManager();
+        // Find Organisation members
+        $orgMembers = $repository->findOneByName($organizationName)->getMemberUsers();
+        // Queue notifications to each member
+        $notification = new Notification();
+        foreach ($orgMembers as $recipient) {
+            // Prevent mailing the currentUser
+            if ($recipient !== $this->getUser()) {
```

Comment:
Wouldn't this mean that the user requesting to join an org is already part of it? I think this check can be removed.